### DISCUSSION TITLE
How to create KEY?

### DISCUSSION THREAD

**Discussion by amgschnappi** [simulot/immich-go#230] [Q&A] [answered]
Hello experts, 
How to create the KEY needed for this tool? Searching on net didnt help much. (https://duckduckgo.com/?q=how+to+create+key+for+immich-go&t=ftsa&atb=v422-1&ia=web)
Please help.
Rgds

**Comment by simulot** [comment, marked as answer]
You can give look at the immich documentation.
Here is the page you are searching for https://immich.app/docs/features/command-line-interface#obtain-the-api-key

**Reply by rementis** [reply]
In the immich web interface, click on your user in top right, then Account Settings.
